content/authorization/tls-guidance.md

```diff
@@ -0,0 +1,71 @@
+---
+title: TLS Usage Guidance
+layout: authorization
+---
+
+# Guidance for the use of TLS with Cerner Ignite
+
+## Introduction
+
+All HL7 FHIR and HL7 SMART endpoints offered by Cerner utilize the Secure HyperText Transfer Protocol ("https") to ensure the confidentiality and integrity of data transmissions.  Cerner strongly __RECOMMENDS__ that application developers also utilize https for protecting their own application endpoints.
```

Comment:
```suggestion
All HL7 FHIR<sup>®</sup> and HL7 SMART<sup>®</sup> endpoints offered by Cerner utilize the Secure HyperText Transfer Protocol ("https") to ensure the confidentiality and integrity of data transmissions.  Cerner strongly __RECOMMENDS__ that application developers also utilize https for protecting their own application endpoints.
```

Reply:
I went ahead and added registered trademarks to all HL7 and SMART references.